go.mod

```diff
@@ -3,17 +3,23 @@ module github.com/alphagov/pay-cli
 go 1.13
 
 require (
+	github.com/Songmu/prompter v0.4.0
+	github.com/aws/aws-sdk-go v1.35.32
 	github.com/briandowns/spinner v1.10.0
+	github.com/go-openapi/strfmt v0.19.11 // indirect
 	github.com/google/go-github v17.0.0+incompatible
 	github.com/google/go-github/v31 v31.0.0
 	github.com/google/uuid v1.1.1
 	github.com/gorilla/schema v1.1.0
+	github.com/jedib0t/go-pretty v4.3.0+incompatible
 	github.com/logrusorgru/aurora v0.0.0-20200102142835-e9ef32dff381
+	github.com/mattn/go-runewidth v0.0.9 // indirect
 	github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4
+	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/viper v1.6.2
 	github.com/tidwall/pretty v1.0.1
 	github.com/urfave/cli/v2 v2.2.0
-	golang.org/x/crypto v0.0.0-20200323165209-0ec3e9974c59
-	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
+	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b
 	golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be
 )
```

Comment:
I take it we don't need the `ginkgo` and `gomega` test dependencies listed in the other repo?

Reply:
Ah - the tests haven't been ported over. Is it possible to do that? Seems a shame to leave them behind 😸

Reply:
👀  +1, some of this was hurriedly put together in a firebreak and introducing a testing structure would be great to make that the bar for future stuff. There's a long list of todos somewhere to make the current stuff ship shape

Reply:
I can look at bringing them over. They aren't much, but I guess it can serve as a potential example of _"what we should strive for"_ going forward.

Reply:
Added them in and edited the README to show how to run them.